### DISCUSSION TITLE
Unexpected colouring changes?

### DISCUSSION THREAD

**Discussion by ilg-ul** [mdx-js/mdx#2520] [Q&A] [answered]
I noticed that colouring starts on blue and turns to red at a certain point.

For example:

![Screenshot 2024-08-03 at 20 30 09](https://github.com/user-attachments/assets/0c6581b5-d636-41a1-8242-9c17390a8251)

My experience with MDX is poor, so sorry if this is a stupid question, but... is this the expected behaviour?

**Comment by remcohaszing** [comment, marked as answer]
See https://github.com/wooorm/markdown-tm-language/issues/10#issuecomment-2265019049

**Reply by ilg-ul** [reply]
Installing the YAML extension fixed the problem.

thank you!
